colors/nord.vim

```diff
@@ -646,6 +646,37 @@ hi! link jsNoise Delimiter
 hi! link jsPrototype Keyword
 hi! link jsRegexpString SpecialChar
 
+" TypeScript
+" > HerringtonDarkholme/yats.vim
+call s:hi("typescriptDecorator", s:nord12_gui, "", s:nord12_term, "", "NONE", "")
+hi! link typescriptOperator Operator
+hi! link typescriptBinaryOp Operator
+hi! link typescriptAssign Operator
+hi! link typescriptMember Identifier
```

Comment:
Sadly the `typescriptMember` group is used for both variables and (class) functions so there's no way to style functions in classes with `nord8` while using `nord4` for variables 😞

Reply:
Yeah, I also encountered this peculiarity.